.travis.yml

```diff
@@ -0,0 +1,46 @@
+sudo: false
```

Comment:
I think it makes sense to retain control over this file within each project. It is pretty generic and most projects will be 90% the same but I don't think that portion will change, it will be the commands that this runs that change and those are already shared.

If we need to add something to all of our projects it's easy enough to add a line in here or mass override the file and resolve the differences by hand (or with `git add -p` most likely).